From 74beba7c8edee65e123b034cb2a28515f8e22eb8 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 11 Feb 2021 16:42:49 -0800
Subject: [PATCH] llvm: Recognize yoe and poky as OE distro

This helps in making right detection for OE built gcc toolchains

The //CLANG_EXTRA_OE_VENDORS_CASES string is replaced with list of
additional Ceses based on CLANG_EXTRA_OE_VENDORS variable in
recipes-devtools/clang/llvm-project-source.inc:add_more_target_vendors()

Signed-off-by: Khem Raj <raj.khem@gmail.com>
Signed-off-by: Martin Jansa <martin.jansa@gmail.com>

Add wrs
Signed-off-by: Yi Fan Yu <yifan.yu@windriver.com>
---
 llvm/lib/Support/Triple.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/llvm/lib/Support/Triple.cpp b/llvm/lib/Support/Triple.cpp
index 4f483c965282..abd181759786 100644
--- a/llvm/lib/Support/Triple.cpp
+++ b/llvm/lib/Support/Triple.cpp
@@ -490,6 +490,6 @@ static Triple::VendorType parseVendor(StringRef VendorName) {
     .Case("mesa", Triple::Mesa)
     .Case("suse", Triple::SUSE)
-    .Case("oe", Triple::OpenEmbedded)
+    .Case("oe", Triple::OpenEmbedded)//CLANG_EXTRA_OE_VENDORS_CASES
     .Default(Triple::UnknownVendor);
 }
 
